localization/yabloc/yabloc_pose_initializer/srv/Semseg.srv

```diff
@@ -0,0 +1,3 @@
+sensor_msgs/Image src_image
```

Comment:
Maybe "SemSeg" is better? Or I think it also could be "SemanticSegmentation.srv"

Reply:
You are right!  I renamed it to `SemanticSegmentation.srv` in 743bfff